schema/release-schema.json

```
@@ -685,7 +685,7 @@
         },
         "items": {
           "title": "Items contracted",
-          "description": "The goods, services, and any intangible outcomes in this contract. Note: If the items are the same as the award do not repeat.",
+          "description": "The goods, services, and any intangible outcomes in this contract. Note: you should not repeat the items if they are the same as the award.",
```

Write from a neutral perspective, rather than a publisher perspective. Here, I suggest using the passive rather than the active voice:

```suggestion
          "description": "The goods, services, and any intangible outcomes in this contract. If the items contracted are identical to the items awarded, this field should be omitted.",
```